package.json

```diff
@@ -26,7 +26,7 @@
     "prepare": "npm run build"
   },
   "engines": {
-    "node": ">=7.6.0"
+    "node": ">=8.9.0"
```

Comment:
lets make this `>=8.0.0`, the only reason we specifically targetting 7.6.0 was because that was when async/await was introduced.

Reply:
Okay, I will change that. I wasn't sure if I should use 8 or the first LTS version.